MatrixKit/Models/Room/MXKRoomDataSource.m

```diff
@@ -1790,6 +1804,9 @@ - (void)processQueuedEvents:(void (^)())onComplete
         // Lock on `eventsToProcessSnapshot` to suspend reload or destroy during the process.
         @synchronized(eventsToProcessSnapshot)
         {
+            // Reset the previous snapshot
+        	bubblesSnapshot = nil;
```

Comment:
bubbles snapshot should be reseted after use (see below bubbles= bubblesSnapshot;)